packages/docusaurus/src/server/i18n.ts

```diff
@@ -25,6 +25,7 @@ export function getDefaultLocaleConfig(locale: string): I18nLocaleConfig {
     label: getDefaultLocaleLabel(locale),
     direction: getLangDir(locale),
     htmlLang: locale,
+    baseUrl: '',
```

Comment:
Base URL can't be empty. If you mean to override it later, use `undefined`, not an empty string.